telegram/ext/picklepersistence.py

```diff
@@ -50,9 +50,10 @@ class PicklePersistence(BasePersistence[UD, CD, BD]):
 
     .. versionchanged:: 14.0
         The parameters and attributes ``store_*_data`` were replaced by :attr:`store_data`.
+        The parameter and attribute ``filename`` were replaced by :attr:`filepath.
```

Comment:
```suggestion
        The parameter and attribute ``filename`` were replaced by :attr:`filepath`.
```

nitpicking: Can you make this an itemized list?

Reply:
what do you mean by 'itemized list'?

Reply:
```
* something
* like
* this :)
```

renders as 
* something
* like
* this :)

Reply:
oh yes of course, noticed it myself, already done